### DISCUSSION TITLE
How to add Debian 12 DEBs to the OBS repository?

### DISCUSSION THREAD

**Discussion by btravouillon** [cobbler/cobbler#3830] [Q&A] [answered]
The documentation at https://cobbler.github.io/downloads/3.x.x.html provides links to repositories. While the latest release 3.3.6 supports Debian 12, the [repository](https://software.opensuse.org/download/package?package=cobbler&project=systemsmanagement%3Acobbler%3Arelease33)  only provides packages for EOL releases of Debian (10 and 11).

I can't find which GH action is used to add Debian 12 to this repo. Could someone please give me a pointer?

**Comment by SchoolGuy** [comment, marked as answer]
Only maintainers of the repository can add this. Since OBS is a seperate system to GitHub the config for this is not to be found in our repositories. I added the OS, so you should habe repos appearing soon for Debian 12, given that there are no build issues.
